relay-config/src/config.rs

```diff
@@ -1676,6 +1697,24 @@ impl Config {
         Duration::from_secs(self.values.cache.eviction_interval.into())
     }
 
+    /// Returns `true` if the persistent envelope buffer is enabled, `false` otherwise.
+    pub fn cache_persistent_buffer_enabled(&self) -> bool {
+        self.values
+            .cache
+            .persistent_envelope_buffer
+            .as_ref()
+            .map_or(false, |b| b.enabled)
+    }
+
+    /// Returns the path to the persistent envelope buffer file.
+    pub fn cache_persistent_buffer_path(&self) -> PathBuf {
+        self.values
+            .cache
+            .persistent_envelope_buffer
+            .as_ref()
+            .map_or(default_persistent_buffer_path(), |b| b.path.clone())
+    }
```

Comment:
nit: Can we just have a `cache_persistent_buffer()` method that returns a reference to the config struct?